modules/notifications/notifications.php

```diff
@@ -4,1292 +4,1453 @@
-		);
-		$this->module = EditFlow()->register_module( 'notifications', $args );
-		
-	}
-	
-	/**
-	 * Initialize the notifications class if the plugin is enabled
-	 */
-	function init() {
-
-		// Register our taxonomies for managing relationships
-		$this->register_taxonomies();
-
-		// Allow users to use a different user capability for editing post subscriptions
-		$this->edit_post_subscriptions_cap = apply_filters( 'ef_edit_post_subscriptions_cap', $this->edit_post_subscriptions_cap );
-		
-		// Set up metabox and related actions
-		add_action( 'add_meta_boxes', array( $this, 'add_post_meta_box' ) );
-
-		// Add "access badge" to the subscribers list.
-		add_action( 'ef_user_subscribe_actions', array( $this, 'display_subscriber_warning_badges' ), 10, 2 );
-	
-		// Saving post actions
-		// self::save_post_subscriptions() is hooked into transition_post_status so we can ensure usergroup data
-		// is properly saved before sending notifs
-		add_action( 'transition_post_status', array( $this, 'save_post_subscriptions' ), 0, 3 );
-		add_action( 'transition_post_status', array( $this, 'notification_status_change' ), 10, 3 );
-		add_action( 'ef_post_insert_editorial_comment', array( $this, 'notification_comment') );
-		add_action( 'delete_user',  array($this, 'delete_user_action') );
-		add_action( 'ef_send_scheduled_email', array( $this, 'send_single_email' ), 10, 4 );
-		
-		add_action( 'admin_init', array( $this, 'register_settings' ) );
-		
-		// Javascript and CSS if we need it
-		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_admin_scripts' ) );
-		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_admin_styles' ) );	
-
-		// Add a "Follow" link to posts
-		if ( apply_filters( 'ef_notifications_show_follow_link', true ) ) {
-			// A little extra JS for the follow button
-			add_action( 'admin_head', array( $this, 'action_admin_head_follow_js' ) );
-			// Manage Posts
-			add_filter( 'post_row_actions', array( $this, 'filter_post_row_actions' ), 10, 2 );
-			add_filter( 'page_row_actions', array( $this, 'filter_post_row_actions' ), 10, 2 );
-			// Calendar and Story Budget
-			add_filter( 'ef_calendar_item_actions', array( $this, 'filter_post_row_actions' ), 10, 2 );
-			add_filter( 'ef_story_budget_item_actions', array( $this, 'filter_post_row_actions' ), 10, 2 );
+				'settings_help_sidebar' => __( '<p><strong>For more information:</strong></p><p><a href="http://editflow.org/features/notifications/">Notifications Documentation</a></p><p><a href="http://wordpress.org/tags/edit-flow?forum_id=10">Edit Flow Forum</a></p><p><a href="https://github.com/danielbachhuber/Edit-Flow">Edit Flow on Github</a></p>', 'edit-flow' ),
```

Comment:
Minor: We should change the danielbachhuber to automattic here.

Reply:
Changed in 69676fd4e45c74b8609102c2820c9136bce8d350